src/actions/google/drive/sheets/google_sheets.ts

```diff
@@ -58,10 +69,20 @@ export class GoogleSheetsAction extends GoogleDriveAction {
                 }
             } catch (e: any) {
                 this.sanitizeGaxiosError(e)
-                winston.error(`Failed execute for Google Sheets. Error: ${e.toString()}`,
-                    {webhookId: request.webhookId})
+                let error: Hub.Error = Hub.errorWith(
+                    HTTP_ERROR.internal,
+                    ` ${LOG_PREFIX} Failed execute. Error: ${e.toString()}`,
+                )
+
+                if (e.code && e.errors && e.errors[0] && e.errors[0].message) {
+                    error = {...error, http_code: e.code, message: `${HTTP_ERROR.internal.description} ${e.errors[0].message}`}
```

Comment:
We need to have the `log_prefix` appended to the error message here as well, otherwise we won't be able to tell which action is erroring with just one log line.